test/reactionsystem.jl

```diff
@@ -0,0 +1,132 @@
+using Catalyst, ModelingToolkit, OrdinaryDiffEq
+
+sbmlfile = "reactionsystem_01.xml"
+
+@parameters t, k1, c1
+@variables s1, s2, s1s2
+
+# t = Variable(:t)
+# k1 = Num(Sym{ModelingToolkit.Parameter{Real}}(:k1))
+# c1 = Num(Sym{ModelingToolkit.Parameter{Real}}(:c1))
+
+COMP1 = SBML.Compartment("c1", true, 3, 2., "nl") 
+SPECIES1 = SBML.Species("s1", "c1", false, nothing, nothing, (1., "substance"), nothing, true)  # Todo: Maybe not support units in initial_concentration?
+SPECIES2 = SBML.Species("s2", "c1", false, nothing, nothing, nothing, (1., "substance/nl"), false)
+KINETICMATH1 = SBML.MathIdent("k1")
+KINETICMATH2 = SBML.MathApply("*", SBML.Math[
+    SBML.MathIdent("k1"), SBML.MathIdent("s2")])
+REACTION1 = SBML.Reaction(Dict("s1" => 1), nothing, nothing, nothing, nothing, KINETICMATH1)
+REACTION2 = SBML.Reaction(Dict("s2" => -1), nothing, nothing, nothing, nothing, KINETICMATH2)
+MODEL1 = SBML.Model(Dict("k1" => 1.), Dict(), Dict("c1" => COMP1), Dict("s1" => SPECIES1), Dict("r1" => REACTION1), nothing, nothing)  # PL: For instance in the compartments dict, we may want to enforce that key and compartment.name are identical
+MODEL2 = SBML.Model(Dict("k1" => 1.), Dict(), Dict("c1" => COMP1), Dict("s2" => SPECIES2), Dict("r2" => REACTION2), nothing, nothing)
```

Comment:
move everything to testset scope
(btw why SCREAM_CAPS?)

Reply:
In Python it seems to be convention that constants in the global scope are in SCREAM_CAPS. I am using these here throughout the test cases. What is the convention in Julia?

Reply:
same convention.

Reply:
fixed with 44d88f5